Backend/app.py

```diff
@@ -68,6 +69,18 @@ def getDatabaseClient():
     logger.info("Using Local Dev Database")
     return boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
 
+
+def handleOptionsRequest(endpoint):
+  @functools.wraps(endpoint)
+  def optionsDecorator(*args, **kws):
+    if request.method =='OPTIONS':
+      status = 200
+      flaskResponse = ResponseCreation.createEmptyResponse(status)
+      return flaskResponse
```

Comment:
Could this be made `return ResponseCreation.createEmptyResponse(status=200)` to get rid of those two lines above the return?